vite.config.ts

```
@@ -1,13 +1,26 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import { keycloakify } from "keycloakify/vite-plugin";
+import themes from "./themes";
```

ESLint

[error] 'themes' is defined but never used.

(@typescript-eslint/no-unused-vars)